### ISSUE TITLE
Pretrained model of the DarkNet Backbone

### ISSUE THREAD

**Issue by zhiqwang** [zhiqwang/yolov5-rt-stack#44]
## 🚀 Feature

Now a pre-trained model is missing under the module `DarkNet` as following

https://github.com/zhiqwang/yolov5-rt-stack/blob/3171d65d2f30397b1811d755b5c923bfbc530d10/models/darknet.py#L46-L56

The best way is to train a pre-trained model from the ImageNet datasets, but its cost is relatively high. Another alternative is to directly cut the backbone branch of the detection model like

https://github.com/zhiqwang/yolov5-rt-stack/blob/3171d65d2f30397b1811d755b5c923bfbc530d10/utils/updated_checkpoint.py#L8-L16

## Motivation

The pre-trained models in backbone are very helpful for accelerating convergence, but some studies have found that the accuracy obtained from scratch training will be better (supplement literature needed).
